src/stacks-hierarchy/Root_PurchaseOverviewScreen/components/FlexTicketDiscountInfo.tsx

```diff
@@ -125,6 +138,24 @@ export const FlexTicketDiscountInfo = ({userProfiles, style}: Props) => {
   );
 };
 
+const mapToFlexProfileList = (
```

Comment:
Might be a better way to do this instead of doing `for-loop`, please let me know if there are better ways :)

Reply:
Could be solved something like this

```ts
const flexProfileList: FlexProfileList[] = userProfileWithCounts.map(
  (userProfile) => ({
    userProfileKey: userProfile.id,
    userProfileName: getReferenceDataName(userProfile, language),
    offer: userProfile.offer,
  })
);
```

Reply:
This doesn't multiply the amount of user profile based on ticket count though, if we buy multiple adult ticket, there's only one user profile added to the list. We want to multiply the user profile by the count of ticket of that type. e.g., if there are 5 adult tickets purchased, we want 5 adult profiles.

Reply:
This one no longer needed after the design update.